x-pack/packages/ml/data_grid/lib/get_field_format_from_index_pattern.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataView } from '@kbn/data-views-plugin/public';
+import type { FieldFormat } from '@kbn/field-formats-plugin/common';
+
+/**
+ * Utility for returning the FieldFormat from a full populated Kibana index pattern object
```

Comment:
I revisited that utility function and found that we no longer use part of what it's doing. Since it was used in one place only anyway I therefor deleted the whole file and adapted the previous consuming code to do inline what's necessary in [7865a60](https://github.com/elastic/kibana/pull/155530/commits/7865a60455c9d26233a97f81b75aa5d36182633c).